libraries/lib-track/Track.h

```diff
@@ -298,10 +298,13 @@ class TRACK_API Track /* not final */
 
    //! public nonvirtual duplication function that invokes Clone()
    /*!
+    @param shallowCopyAttachments if true, then share AttachedTrackObjects
     @pre `IsLeader()`
     @post result: `NChannels() == result->NChannels()`
     */
-   virtual TrackListHolder Duplicate() const;
+   virtual TrackListHolder Duplicate(bool shallowCopyAttachments = false) const;
+
+   void ReparentAllAttachments();
```

Comment:
`Duplicate or `ReparentAllAttachments`?  Sorry, no, that would break compilation after the last commit.

`TrackListRestorer` uses `Duplicate`.  `PendingTracks` uses both.